.travis.yml

```diff
@@ -23,6 +23,21 @@ matrix:
   allow_failures:
     - env: lt_branch=RC_1_0 gui=true build_system=cmake
     - env: lt_branch=RC_1_0 gui=false build_system=cmake
+  include:
```

Comment:
Please move the block above `allow_failures`.

Reply:
Why?